optax/contrib/cocob.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Backpropagating variant of the COntinuous COin Betting stochastic
```

Comment:
One-line docstring summary should end with ".", ".)", "?", or "!" [g-short-docstring-punctuation]

Reply:
One-line docstring summary should be followed by a blank line [g-no-space-after-docstring-summary]